tests/nodes/utils.ts

```diff
@@ -99,4 +97,18 @@ async function nodesConnect(localNode: PolykeyAgent, remoteNode: PolykeyAgent) {
   } as NodeAddress);
 }
 
-export { generateNodeIdForBucket, incrementNodeId, nodesConnect };
+function generateRandomNodeId(): NodeId {
+  // Make it easy to read with an identifier
+  const base = 'Node-' + Math.trunc(Math.random() * 10000).toString(16) + '-';
+  const random = keysUtils.getRandomBytesSync(32 - base.length);
+  return IdInternal.fromBuffer<NodeId>(
+    Buffer.concat([Buffer.from(base), random]),
+  );
+}
```

Comment:
I think `base` doesn't require the extra random number. You already getting random bytes.

Also this should be in `tests/utils.ts`.

Reply:
Here's a better example:

```
function generateRandomNodeId(): NodeId {
  // Make it easy to read by using hex
  const random = keysUtils.getRandomBytesSync(16).toString('hex');
  return IdInternal.fromBuffer<NodeId>(
    Buffer.from(random),
  );
}
```

Reply:
It has been updated.